### ISSUE TITLE
console.Confirm should uppercase the answer

### ISSUE THREAD

**Issue by kogitant** [aws-cloudformation/rain#18]
Seeing this after answering y is painful:
```
Do you wish to continue? (Y/n) y
User cancelled deployment.
```

I'd propose uppercasing the answer before comparison:
```
if strings.ToUpper(answer) == "Y" || (defaultYes && answer == "") {
		return true
}
```
https://github.com/aws-cloudformation/rain/blob/85305c13e5a904bb735046ae1cc5a9cb4f0bc01f/console/console.go#L78

**Comment by ctd** [comment]
This same change was merged in https://github.com/aws-cloudformation/rain/pull/9

There hasn't been a versioned release between then and now, but you should have the fix if you install rain using `go get` from the master branch.

**Comment by stilvoid** [comment]
Fixed in `v0.8.0`
